resources/build_resources.sh

```diff
@@ -13,6 +13,9 @@
 # The path to output all built .py files to: 
 UI_PYTHON_PATH=../python/tk_multi_publish/ui
 
+# The path to where the PySide binaries are installed
+PYTHON_BASE="/Applications/Shotgun.app/Contents/Resources/Python"
```

Comment:
+1 not to hard code this. IMHO Seems we are overcomplicating things a bit. @josh-t suggestion to put add a comment about it seems better. And if we do update it, we should probably not do it in a single app, but instead look at if we can centralize this build script that is present in so many apps. or do a blanket update.